### PR TITLE
EWPP-1864: Style the lists with illustrations paragraphs

### PR DESCRIPTION
**Pull request by yenyasinn**

### CODE REVIEW

File: oe_theme.theme [hunk 1]
```diff
@@ -1711,3 +1713,143 @@ function oe_theme_preprocess_oe_theme_helper_site_navigation(&$variables) {
...
   }
 }
+
+/**
+ * Process common fields of illustration list paragraphs.
+ *
+ * @param array $variables
+ *   Render array.
+ */
+function _oe_theme_preprocess_paragraph_oe_illustration_list(array &$variables) {
```
Comment (sergepavle) [inline oe_theme.theme]: Missed return type

File: oe_theme.theme [hunk 2]
```diff
@@ -1711,3 +1713,143 @@ function oe_theme_preprocess_oe_theme_helper_site_navigation(&$variables) {
...
+    // Get sub-paragraph translation.
+    $language_id = $paragraph->language()->getId();
+    $sub_paragraph = $entity_repository->getTranslationFromContext($sub_paragraph, $language_id);
+    // Get the image from attached media.
+    $image = [];
+    $media = _oe_theme_prepare_media($sub_paragraph->get('field_oe_media')->entity, $variables['user'], $cacheability, $language_id);
+    if ($media) {
+      $source = $media->getSource();
+      if ($source instanceof Image && ($file_entity = $media->get('oe_media_image')->entity)) {
+        $uri = $file_entity->getFileUri();
```
Comment (sergepavle) [inline oe_theme.theme]: Can be used directly \Drupal\file\Entity\File::createFileUrl

File: oe_theme.theme [hunk 3]
```diff
@@ -1711,3 +1713,143 @@ function oe_theme_preprocess_oe_theme_helper_site_navigation(&$variables) {
...
+  _oe_theme_preprocess_paragraph_oe_illustration_list($variables);
+  $variables['items'] = [];
+
+  $paragraph = $variables['paragraph'];
+  $ratio = $paragraph->get('field_oe_illustration_ratio')->first()->value;
+  $builder = \Drupal::entityTypeManager()->getViewBuilder('paragraph');
+  /** @var \Drupal\paragraphs\Entity\Paragraph $sub_paragraph */
+  foreach ($paragraph->get('field_oe_paragraphs')->referencedEntities() as $sub_paragraph) {
+    // Get sub-paragraph translation.
+    $sub_paragraph = \Drupal::service('entity.repository')
```
Comment (sergepavle) [inline oe_theme.theme:1781]: Is it safe always expect paragraph as referenced entity?

File: oe_theme.theme [hunk 4]
```diff
@@ -1711,3 +1713,143 @@ function oe_theme_preprocess_oe_theme_helper_site_navigation(&$variables) {
...
+}
+
+/**
+ * Implements hook_preprocess_paragraph().
+ */
+function oe_theme_preprocess_paragraph__oe_illustration_list_images(array &$variables): void {
+  _oe_theme_preprocess_paragraph_oe_illustration_list($variables);
+
+  $paragraph = $variables['paragraph'];
+  $ratio = $paragraph->get('field_oe_illustration_ratio')->first()->value;
```
Comment (sergepavle) [inline oe_theme.theme]: Seems like don't need it here.

File: templates/paragraphs/paragraph--oe-illustration-list-flags.html.twig [hunk 5]
```diff
@@ -0,0 +1,20 @@
...
+ */
+#}
+{% if title is not empty %}
+  <h2 class="ecl-u-type-heading-2">
+    {{ title }}
+  </h2>
+{% endif %}
+
+{{ pattern('list_with_illustration', {
+  'variant': variant,
```
Comment (sergepavle) [inline templates/paragraphs/paragraph--oe-illustration-list-flags.html.twig]: I don't see any more variants in the pattern. Do we need that?

File: templates/paragraphs/paragraph--oe-illustration-list-icons.html.twig [hunk 6]
```diff
@@ -0,0 +1,20 @@
...
+ */
+#}
+{% if title is not empty %}
+  <h2 class="ecl-u-type-heading-2">
+    {{ title }}
+  </h2>
+{% endif %}
+
+{{ pattern('list_with_illustration', {
+  'variant': variant,
```
Comment (sergepavle) [inline templates/paragraphs/paragraph--oe-illustration-list-icons.html.twig]: I don't see any more variants in the pattern. Do we need that?

File: templates/paragraphs/paragraph--oe-illustration-list-images.html.twig [hunk 7]
```diff
@@ -0,0 +1,20 @@
...
+ */
+#}
+{% if title is not empty %}
+  <h2 class="ecl-u-type-heading-2">
+    {{ title }}
+  </h2>
+{% endif %}
+
+{{ pattern('list_with_illustration', {
+  'variant': variant,
```
Comment (sergepavle) [inline templates/paragraphs/paragraph--oe-illustration-list-images.html.twig]: I don't see any more variants in the pattern. Do we need that?

File: tests/src/Kernel/fixtures/rendering.yml [hunk 8]
```diff
@@ -1,2961 +1,10 @@
-- array:
```
Comment (sergepavle) [inline tests/src/Kernel/fixtures/rendering.yml]: Why such big number of tests has been removed? Is it accidental removal?

File: oe_theme.theme (hunk 3 above)
Comment (imanoleguskiza) [reply oe_theme.theme:1781]: I would say so, we are targetting a specific field of a specific paragraph type, so we can safely assume what will be in it.

**Review by sergepavle — DISMISSED** [review]

**Comment by yenyasinn** [comment]
Closed in favour https://github.com/openeuropa/oe_theme/pull/1017

**Pull request closed without merging**